### ISSUE TITLE
Is there a way to reduce thirst decreasing rate?

### ISSUE THREAD

**Issue by Maladik40** [ghen-git/Thirst-Mod#115]
Hi, I'm doing a modpack with this mod but I have to change the decreasing rate of thirst cuz it's pretty unbalanced for my modpack. Is there a way to do it?

**Comment by ghen-git** [comment]
yes! inside the common config, look for an option called `thirstDepletionModifier`
